packages/CollabCodeEdit-Tests.package/CCEMainModelTest.class/instance/testNoSessionLoadedAndServerConnectionEstablished.st

```diff
@@ -0,0 +1,20 @@
+tests backend
+testNoSessionLoadedAndServerConnectionEstablished
+	model client connected: true.
+	model session status setToNoRemoteSession.
+	
+	self assert: model serverConnectionAndNoRemoteSession.
+	
+	model client connected: false.
+	
+	self deny: model serverConnectionAndNoRemoteSession.
+	
+	model client connected: true.
+	model session status setToCleanRemoteSession.
+	
+	self deny: model serverConnectionAndNoRemoteSession.
+	
+	model client connected: false.
+	model session status setToCleanRemoteSession.
+	
+	self deny: model serverConnectionAndNoRemoteSession.
```

Comment:
I have the feeling, that those are 4 distinct test cases. Is there a reason to merge them?

Reply:
it tests one method.